lib/Runtime/Debug/DiagHelperMethodWrapper.cpp

```diff
@@ -212,8 +212,7 @@ namespace Js
         // Note: there also could be plain OutOfMemoryException and StackOverflowException, no special handling for these.
         if (!exceptionObject->IsDebuggerSkip() ||
             exceptionObject == scriptContext->GetThreadContext()->GetPendingOOMErrorObject() ||
-            exceptionObject == scriptContext->GetThreadContext()->GetPendingSOErrorObject() ||
-            !scriptContext)
```

Comment:
I have verified all the call sites of HandleHelperOrLibraryMethodWrapperException and they have dependency on scriptcontext being valid. Crash would be much earlier if scriptcontext was null. I don't see how this check is useful. Won't fix for me.